getting-started/explore-control-features-sl.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Explore Control Features
+page_title: Explore Control Features
+description: Explore Control Features
+slug: getting-started-explore-control-features-sl
+tags: Explore Control Features
+published: True
+position: 2
+---
+
+# Explore Control Features
+
+Once you have the controls [working in your project]({%slug getting-started-first-steps-sl%}), it's time to see what they can do. This article provides a short overview of how to get started with finding control functionality so you can use it.
+
+## Demos
+
+To get an overview of what each control offers, the fastest approach is to go to our [Live Demos](http://demos.telerik.com/silverlight).
+
+You can also [download the source code of the demo application]({%slug download-product-files%}) from your telerik.com account as a Visual Studio project and play around with the demos in the comfort of your IDE. The `Telerik_UI_for_WPF_[version]_Demos.zip` archive contains the respective source code.
+
+## Explore properties and tags
+
+You can **explore** the available properties, events and inner tags of any control through the VS **intellisense** to get a quick glimpse of their concepts and availability.
+
+>caption Figure 1: Explore Intellisense in the markup
+
+![Explore Intellisense in the markup](adding-telerik-ui-to-an-application/images/installation-adding-application-using-intellisense-vs-wpf-1.png "Explore Intellisense in the markup")
```

Comment:
This picture points to a file that contains "wpf" in its name. Is this the correct file?